src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -189,12 +193,18 @@ trait ClusterComponent extends LeoComponent {
         ClusterStatus.withName(clusterRecord.status),
         clusterRecord.hostIp map IP,
         clusterRecord.createdDate.toInstant,
-        clusterRecord.destroyedDate map { _.toInstant },
+        getDestroyedDate(clusterRecord.destroyedDate),
         labels,
         clusterRecord.jupyterExtensionUri flatMap { GcsPath.parse(_).toOption }
       )
     }
 
+    private def getDestroyedDate(destroyedDate:Timestamp): Option[Instant] = {
```

Comment:
I think this (and the sentinel magic-value destroyed date) should probably be functions on `object Cluster` instead -- but there might be disagreement here

Reply:
🤷‍♂️ I like it here -- in `Cluster` we abstract away the fact that we have a magic sentinel value in the database. The sentinel-handling code is private to `ClusterComponent`.

Reply:
I prefer this here as well, for the same reason as above.